src/js/files/querys.ts

```diff
@@ -1,11 +1,13 @@
-import { useMutation, useQuery } from "react-query";
+import { useInfiniteQuery, useMutation, useQuery } from "react-query";
 import { Request } from "../app/request";
-import { FileType } from "./types";
+import { FileResponse, FileType } from "./types";
 
 export const fileQueryKeys = {
     all: () => ["files"] as const,
     lists: () => ["files", "list"] as const,
     list: (type: string, filters: Array<string | number | boolean>) => ["files", "list", type, ...filters] as const,
+    infiniteList: (type: string, filters: Array<string | number | boolean>) =>
+        ["files", "infiniteList", type, ...filters] as const,
```

Comment:
You mighta duped my bad code here... Should be:

```suggestion
    infiniteList: (type: string, filters: Array<string | number | boolean>) =>
        ["files", "list", "infinite" type, ...filters] as const,
```
That way you can reset all types of list at once say when WS updates come in without having to worry about list type